### PR TITLE
Twl09008181 proposal submission

### PR DESCRIPTION
**Pull request by Twl09008181**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/Twl09008181/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+I think it's worthy to speed up kmeans by C++, and make it a general usage for python programmer.     
+The interface of kmeans is simple, we can just use 2D array to represent the dataset.     
+It's easy to get dataset from sklearn.    
+
+
+## Goal
+
+1. Implment a C++ version KMeans.
+2. Try to speed up with parallel programming skill.
```
Comment (yungyuc) [inline project/Twl09008181/README.md:52]: Let's see the scaling factor you will get.

File: project/Twl09008181/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+
+## Schedule   
+
+week0(11/01): sequential kmeans algorithm study.          
+week1(11/07): Implement sequential kmeans by C++ and try to use pybind to call by python.   
+week2(11/14): parallel kmeans algorithm study.      
+week3(11/21): parallel programming framework study.     
+week4(11/28): Implement paralle kmeans by C++.      
+week5(12/05): Testing and compare with sklearn kmeans.
```
Comment (yungyuc) [inline project/Twl09008181/README.md:88]: Testing sooner will save you time.

**Review by yungyuc — APPROVED** [review]
LGTM.  Leave a message to let me know when you want it to be merged.

**Comment by Twl09008181** [comment]
> LGTM. Leave a message to let me know when you want it to be merged.

Thanks for reviewing my proposal, I think it is ok to be merged.

**Pull request merged**